apps/storybook/src/components/ai-chat-input/index.tsx

```
@@ -0,0 +1,163 @@
+              </motion.div>
+            )}
+          </AnimatePresence>
+        </div>
+      </div>
+    );
+  }
+);
+
+AIChatInput.displayName = 'AIChatInput';
+
+export { AIChatInput, chatInputVariants };
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)